backend/src/main/java/de/neuefische/backend/model/openai/ChatGptMessage.java

```diff
@@ -0,0 +1,31 @@
+package de.neuefische.backend.model.openai;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ChatGptMessage {
+    /**
+     {
+     "messages":
+     [
+     {
+     "role": "user",
+     "content": "Say this is a test!"
+     }
+     ]
+     }
+     */
```

Comment:
you dont want comments in your production code

Reply:
done!